fe/fe-core/src/main/java/org/apache/iceberg/MetadataParser.java

```diff
@@ -265,7 +265,10 @@ private ContentFile<?> parseThriftToIcebergDataFile(TMetadataEntry entry) {
                     equalityFieldIds,
                     sortId,
                     splitOffsets,
-                    keyMetadata
+                    keyMetadata,
+                    null,
```

Comment:
looks like we need to add these paramters to support dv?

Reply:
Yes. But we do not support read delete-vectors in this PR, right? 
We will fully support V3 spec in other PRs, WDYT?

Reply:
Yes，we can support read iceberg dv later, we just keep minimum changes in this PR.